projects/igniteui-angular/src/lib/accordion/accordion.component.ts

```diff
@@ -387,7 +387,9 @@ export class IgxAccordionComponent implements AfterContentInit, AfterViewInit, O
             });
             panel.contentCollapsing.pipe(takeUntil(this._unsubChildren$)).subscribe((args: IExpansionPanelCancelableEventArgs) => {
                 const evArgs: IAccordionCancelableEventArgs = { ...args, owner: this, panel: args.owner };
-                this.panelCollapsing.emit(evArgs);
+                if (!args.owner.closeAnimationPlayer?._started) {
```

Comment:
Do we want to emit `panelCollapsing` if `closeAnimationPlayer` is falsy? I'm asking because right now if the `closeAnimationPlayer` is falsy, this may not behave as expected as the entire thing will resolve to `undefined` and then the negation will make it `true`, so the event will emit. If we want to emit the event only if the closing animation hasn't started then it might be a good idea to separate the check to be something like `args.owner.closeAnimationPlayer && !args.owner.closeAnimationPlayer._started`.

Reply:
When a panel starts collapsing the closeAnimationPlayer is still null and the event should be emitted, however, if the check is `args.owner.closeAnimationPlayer && !args.owner.closeAnimationPlayer._started` the event would not be emitted.